cypress/fixtures/selectors.json

```diff
@@ -1,30 +1,30 @@
 {
-  "clearValues": ".react-select__clear-indicator",
-  "disabledCheckbox": "#cypress-container-single #cypress-single__disabled-checkbox",
-  "firstMultiValueRemove":
-    "#multi-select .react-select__multi-value__remove:first",
-  "groupColor": "#cypress-container-single #cypress-single-grouped .react-select__group",
-  "menuGrouped": "#grouped-options-single .react-select__menu",
-  "menuMulti": "#multi-select .react-select__menu",
+  "singleBasic": "#cypress-single",
+  "singleBasicSelect": "#basic-select-single",
+  "singleClearable": "#cypress-single-clearable",
+  "singleClearableSelect": "#clearable-select-single",
+  "singleGroupedSelect": "#grouped-options-single",
```

Comment:
This section is for the various types of single select components (and their containers) that we'll want to make assertions about.